presto-docs/src/main/sphinx/connector/elasticsearch.rst

```diff
@@ -185,6 +185,51 @@ Elasticsearch Presto
 (all others)  (unsupported)
 ============= =============
 
+Array Types
+^^^^^^^^^^^
+
+Elasticsearch has `no dedicated array type <https://www.elastic.co/guide/en/elasticsearch/reference/current/array.html>`_ so the `_meta field <https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-meta-field.html>`_ of the index is used to store an property structure that contains
+the ``isArray`` property to specify which fields in the mapping are to be treated as array types.
+
+Example:
```

Comment:
For example, if you can have an Elasticsearch index that contains documents with the following structure: